### PR TITLE
Add autofix to selector-combinator-space-after

### PR DESCRIPTION
**Pull request by ota-meshi**
> Which issue, if any, is this issue related to?

`selector-combinator-space-after` in #2829

> Is there anything in the PR that needs further explanation?

ex.

* option: `always`

    code:
    ```css
    a >.b {}
    ```
    fixed:
    ```css
    a > .b {}
    ```

* option: `never`

    code:
    ```css
    a > .b {}
    ```

    fixed:
    ```css
    a >.b {}
    ```

### CODE REVIEW

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 1]
```diff
@@ -135,79 +136,136 @@ testRule(rule, {
     {
       code: "namespace|type#id > .foo {}",
       description: "qualified id with namespace"
+    },
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: I'm confused by these tests are they aren't valid CSS. Do we need them?

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 2]
```diff
@@ -135,79 +136,136 @@ testRule(rule, {
...
+    {
+      code: "a/*comment*/>/*comment*/a, b/*comment*/>/*comment*/b {}",
+      fixed: "a/*comment*/> /*comment*/a, b/*comment*/> /*comment*/b {}",
+      description: "comment",
+      message: messages.expectedAfter(">"),
+      line: 1,
+      column: 13
+    },
+    {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: Can you move this and the following tests into the `syntax: scss` `testRule` block at the bottom of the file as nesting without the _nesting selector_ (`&`) isn't in the draft spec?

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 3]
```diff
@@ -315,87 +373,131 @@ testRule(rule, {
...
+      code: "a/*comment*/ > /*comment*/a, b/*comment*/ > /*comment*/b {}",
+      fixed: "a/*comment*/ >/*comment*/a, b/*comment*/ >/*comment*/b {}",
+      description: "comment",
+      message: messages.rejectedAfter(">"),
+      line: 1,
+      column: 14
+    },
+    {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: Can you move this and the following tests into the `syntax: scss` `testRule` block at the bottom of the file, please?

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 4]
```diff
@@ -415,8 +518,61 @@ testRule(rule, {
   reject: [
     {
       code: "a+  a {}",
+      fixed: "a+ a {}",
       description: "two spaces after + combinator",
       message: messages.expectedAfter("+")
     }
   ]
 });
+testRule(rule, {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: Empty line above, please.

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 5]
```diff
@@ -415,8 +518,61 @@ testRule(rule, {
...
+    {
+      code: ".a { &+.b {} }",
+      fixed: ".a { &+ .b {} }",
+      message: messages.expectedAfter("+")
+    }
+  ]
+});
+
+// cssnext
+testRule(rule, {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: Can you move the tests in this block into the first `testRule` block (the one without any `syntax: property`), and then remove this `testRule` block, please?

The syntax [is valid draft spec](https://tabatkins.github.io/specs/css-nesting/#direct) so [belongs](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#best-practices) in the first `testRule`.

File: lib/rules/selector-combinator-space-after/__tests__/index.js [hunk 6]
```diff
@@ -415,8 +518,61 @@ testRule(rule, {
...
     }
   ]
 });
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: ["always"],
+  fix: true,
+
+  accept: [
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-after/__tests__/index.js]: Can you move the tests in this block into the first `testRule` block (the one without any `syntax: property`), please?

The syntax is valid draft spec so [belongs](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#best-practices) in the first `testRule`.

File: lib/rules/selector-combinator-space-before/__tests__/index.js [hunk 7]
```diff
@@ -143,6 +143,18 @@ testRule(rule, {
     {
       code: "namespace|type#id > .foo {}, space|customtype#id_withunder > a {}",
       description: "qualified id with namespace selector list"
+    },
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-before/__tests__/index.js]: Can you move this tests into their own `testRule` with `syntax: scss` as they are without a nesting selector?

File: lib/rules/selector-combinator-space-before/__tests__/index.js [hunk 8]
```diff
@@ -209,6 +221,13 @@ testRule(rule, {
       message: messages.expectedBefore(">>>"),
       line: 1,
       column: 2
+    },
+    {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-before/__tests__/index.js]: Move to `testRule` with `syntax: scss`.

File: lib/rules/selector-combinator-space-before/__tests__/index.js [hunk 9]
```diff
@@ -313,6 +332,18 @@ testRule(rule, {
     {
       code: "namespace|type#id> .foo {}",
       description: "qualified id with namespace"
+    },
+    {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-before/__tests__/index.js]: Move to `testRule` with `syntax: scss`.

File: lib/rules/selector-combinator-space-before/__tests__/index.js [hunk 10]
```diff
@@ -400,6 +431,13 @@ testRule(rule, {
       message: messages.rejectedBefore(">>>"),
       line: 1,
       column: 3
+    },
+    {
```
Comment (jeddy3) [inline lib/rules/selector-combinator-space-before/__tests__/index.js]: Move to `testRule` with `syntax: scss`.

File: lib/rules/selector-combinator-space-after/__tests__/index.js (hunk 1 above)
Comment (ota-meshi) [reply lib/rules/selector-combinator-space-after/__tests__/index.js]: I didn't know it was invalid css...
I removed this test case.

**Comment by ota-meshi** [comment]
@jeddy3 Thank you for reviews and many comments!
I changed it, so please check it again.

**Comment by jeddy3** [comment]
> -   Added: `selector-combinator-space-after` autofix ([#3446](https://github.com/stylelint/stylelint/pull/3446)).

**Pull request merged**